mobile/src/components/course/common/constants/constants.ts

```diff
@@ -0,0 +1,5 @@
+enum CourseDescriptionFirstCharsCount {
+  DEFAULT_COUNT = 200,
+}
```

Comment:
We won't have more that one value here, constant is enough for this
```suggestion
const COURSE_SHORT_DESCRIPTION_LENGTH = 200;
```